src/30_Advanced/Amp-Carousel_with_Amp-List.html

```diff
@@ -0,0 +1,94 @@
+<!---{
+  "preview": "default"
+  }--->
+<!--
+  #### Introduction
+  AMP users might want to pull data from an endpoint to display in a carousel. This example incoporates a dynamic amp-carousel as well as showing how to not only rely on the images served from the endpoint
+   
+-->
+<!-- -->
+
+
+<!doctype html>
+<html ⚡>
+ <head>
+   <meta charset="utf-8">
+   <link rel="canonical" href="hello-world.html">
+   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+   <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+   <script async src="https://cdn.ampproject.org/v0.js"></script>
+   <!-- #### Setup -->
+  <!--
+    Import the `amp-list` and `amp-mustache` and `amp-carousel` and `amp-bind` components.
```

Comment:
Can you split the import comment so that each extension import has a comment (this way all imports will be shown in the generated doc page). Similar to here: https://github.com/ampproject/amp-by-example/blob/master/src/40_Dynamic_AMP/Multi_Page_Flow.html#L22

Reply:
Will fix this in my next commit